support-frontend/assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -17,6 +17,7 @@ export const pageUrlRegexes = {
   subscriptions: {
     subsShowcaseAndDigiSubPages: '(/??/subscribe(\\?.*)?$|/??/subscribe/digital(\\?.*)?$)',
     digiSubLandingPages: '(/??/subscribe/digital/gift(\\?.*)?$|/??/subscribe/digital(\\?.*)?$)',
+    digiSubLandingPagesNotGift: '(/??/subscribe/digital(\\?.*)?$)',
```

Comment:
We need this regex to include the checkout as well or we don't track the abtest participations correctly in the datalake acquisitions table

Reply:
I've decided to overhaul the regexes a bit and use actual regex literals rather than strings (mostly because I find them easier to compose and test). The new one should explicitly include the checkout as well.